### ISSUE TITLE
Rubrica General

### ISSUE THREAD

**Issue by alexvilanovab** [alexvilanovab/ML-Speech-Gender-Recognition#20]
- [x] **Identificación del problema:** el tipo de problema a resolver se se identifica correctamente y resuelve un problema concreto y bien definido.
- [x] **Análisis del problema:** el análisis del dataset es adecuado y se justifica porque la variable objetivo está correlacionada con el resto de características. Se han usado todos los plots necesarios para la justificación.
- [x] **Limpieza del dataset:** se eliminan todos los NaN del dataset sin perder información relevante infiriendo de la forma más precisa posible.
- [x] **Características nuevas:** se han extraído características nuevas cuando era necesario y se ha comprobado que ayudan a mejorar la precisión final.
- [x] **Análisis del dataset:** se usan técnicas que explican la varianza de los datos y se comprueba que características no aportan nada.
- [x] **Seleccion de caracteristicas:** se ha justificado la selección de características para predecir la variable objetivo y se ha comprobado usando algún algoritmo de feature selection diferente a PCA.
- [x] **Análisis de componentes principales:** se ha aplicado un método de reducción de dimensionalidad y se analiza cuánta varianza puede llegar a explicar cada componente. Además se comprueba los efectos de transformar los datos sobre un modelo de ML comparando contra los datos originales.
- [x] **Modelos de ML:** se ha probado más de un modelo de ML y son adecuados para resolver el problema. Se han ajustado los hyperparameters correctamente. Ha tenido en cuenta usar CV y se ha validado con un juego de datos de test una vez obtenido el modelo final. No hay signos de overfitting ni underfitting.
- [x] **Análisis de las predicciones obtenidas:** se justifica correctamente el accuracy obtenido gracias a todo el trabajo previo.
- [x] **Clustering:** en el caso de un problema de clasificación se han analizado los posibles clusters que formaban el problema y si se corresponden con el número de etiquetas propuesto por el dataset. Se ha utilizado alguna métrica como el coste o calinski para justificar el número de clusters obtenidos.
- [x] **SVM (modelo obligatorio):** Se han probado los kernels lineal, polinómico, RBF y se ha ejecutado un grid search para encontrar la configuración de parámetros que mejor accuracy proporciona.
- [x] **Neural Network (modelo obligatorio):** se ha creado una arquitectura que permite con los parámetros de loss, optimización y selección de la función de activación de la última capa obtener una accuracy similar o mejor a los modelos de ML tradicionales y están bien justificados.
